dftimewolf/lib/collectors/grr_base.py

```diff
@@ -101,9 +107,13 @@ def _WrapGRRRequestWithApproval(
           self.ModuleError(message, critical=True)
 
         # Otherwise, send a request for approval
-        grr_object.CreateApproval(
+        approval = grr_object.CreateApproval(
             reason=self.reason, notified_users=self.approvers)
         approval_sent = True
+        approval_url = ('{0:s}/#/users/{1:s}/approvals/client/{2:s}/{3:s}'.
+                        format(self.grr_url, os.environ['USER'],
```

Comment:
I think you can use `approval.username` here instead of `os.environ['USER']`

Reply:
ack!